client/html/layouts/partials/_head.html

```diff
@@ -25,6 +25,7 @@
     unlock_next: "ON_CORRECT",
     locale: "en",
     audio_recorder_timeout: 100,
+    authoring_tool_preview_settings: ['unlock_next=ON_CORRECT'],
```

Comment:
I think it is reasonable to think that at some point someone will want something changed. And it wasn't hard to add. So I just figured I would add it.